assets/css/map/controls/_layers_control.scss

```
@@ -21,19 +21,53 @@
   }
 }
 
-.c-layers-control__layers_list {
+.c-layers-control__pill {
+  position: absolute;
+  top: -6px;
+  left: -6px;
+  width: 22px;
+  height: 13px;
+  border-radius: 16px;
+
+  background-color: $color-strawberry-500;
+
+  text-align: center;
+  color: $white;
+  font-size: 11px;
+  line-height: 11px;
```

suggestion: The line height is the same as the font size, so we should use unitless values

```suggestion
  line-height: 1;
```

> The used value is this unitless `<number>` multiplied by the element's own font size. The computed value is the same as the specified `<number>`. In most cases, this is the ___preferred___ way to set line-height and avoid unexpected results due to inheritance.
>
> -- https://developer.mozilla.org/en-US/docs/Web/CSS/line-height#number